nameko_tracer/utils.py

```diff
@@ -23,7 +23,7 @@ def serialise_to_string(value):
 
 def safe_for_serialisation(value):
     no_op_types = six.string_types + six.integer_types + (float,)
```

Comment:
I think adding `None` to the list of no-op types would be clearer:

```
no_op_types = six.string_types + six.integer_types + (float, None)
```

Reply:
I tried that and it didn't work, `None` isn't a python type, its `NoneType` , but it seems in python 3 you can't import it anywhere :/
```
TypeError: isinstance() arg 2 must be a type or tuple of types
```

Reply:
looks like `type(None)` works